src/cli/commands/deploy.ts

```diff
@@ -31,6 +32,8 @@ export default function registerCommand(program: Command) {
     .option("-a, --app-location <path>", "the folder containing the source code of the front-end application", DEFAULT_CONFIG.appLocation)
     .option("-i, --api-location <path>", "the folder containing the source code of the API application", DEFAULT_CONFIG.apiLocation)
     .option("-O, --output-location <path>", "the folder containing the built source of the front-end application", DEFAULT_CONFIG.outputLocation)
+    .option("-l, --api-language <apiLanguage>", "the runtime language of the function/api")
+    .option("-v, --api-version <apiVersion>", "the version of the function runtime language")
```

Comment:
`-v` is already used to show the swa version.

![image](https://user-images.githubusercontent.com/1699357/194336175-80c587a5-b47b-4ad9-87c3-5441b54c27d7.png)

I suggest `-av`:

```suggestion
    .option("-av, --api-version <apiVersion>", "the version of the function runtime language")
```